starter-code/scripts/articleView.js

```diff
@@ -73,41 +73,69 @@ articleView.setTeasers = function() {
+function Entry() {
+  this.title = $('#entryTitle').val();
+  this.body = $('#entryBody').val();
+  this.author = $('#entryAuthor').val();
+  this.authorUrl = $('#entryAuthorUrl').val();
+  this.category = $('#entryCategory').val();
+  this.published = $('#entryPublished').val();
+}
+
+articleView.templateAndRender = function(newEntry) {
+  var handlebarsStr = $('#formTemplate').html();
+  var formCompilerFunc = Handlebars.compile(handlebarsStr);
+  var newHtml = formCompilerFunc(newEntry);
+  $('#articles').append(newHtml);
+}
 
-  // TODO: Hide the article-export section on page load
+articleView.initNewArticlePage = function() {
+ // DONE: Make the tabs work. Right now, you're seeing all the tab content (items with a class of tab-content) on the page at once. The section with the id of "write" should show when the "write" tab is clicked; it is also the default and should be shown on page load. The section with the id of "articles" should show when the "preview" tab is clicked.
+// The articleView.handleMainNav function is doing this for us already, so...
+// Time spent: 20 minutes figuring out that nothing was needed...
 
-  $('#article-json').on('focus', function(){
+  // DONE: Hide the article-export section on page load
+  $('#article-export').hide();
+  $('#article-json').on('focus', function(e){
+    e.preventDefault();
     this.select();
   });
 
-  // TODO: Add an event handler to update the preview and the article-export field if any inputs change.
-
-};
+  // DONE: Add an event handler to update the preview and the article-export field if any inputs change.
+  articleView.create();
+}
 
 // this is the function that generates the preview and shows the export field
 articleView.create = function() {
-  // TODO: Set up a var to hold the new article we are creating.
-  // Clear out the #articles element, so we can put in the updated preview
-
-
-  // TODO: Instantiate an article based on what's in the form fields:
-
-
-  // TODO: Use our interface to the Handblebars template to put the article preview into the DOM:
-
-
-  // TODO: The new articles we create will be shown as JSON in an element in our article-export section. From there, we can copy/paste the JSON into our source data file.
+  $('#entryForm').on('change', function(event){
+    event.preventDefault();
+    // Clear out the #articles element, so we can put in the updated preview
+    $('#articles').empty();
+
+    // DONE: Set up a var to hold the new article we are creating.
+    var newEntry = new Entry(this);
+    // DONE: Instantiate an article based on what's in the form fields:
+    // DONE: Use our interface to the Handblebars template to put the article preview into the DOM:
+    //we call the `Entry` constructor, then pass in `this` which is referencing the form
```

Comment:
I see that you create the new entry. But where to you add it to the HTML? What do we need to call to add our new article to the html?